Sources/CoreAardvarkTests/ARKDataArchiveTests.m

```diff
@@ -226,7 +226,7 @@ - (void)test_appendArchiveOfObject_trimsArchive;
         [expectation5 fulfill];
     }];
     
-    [self waitForExpectationsWithTimeout:5.0 handler:nil];
+    [self waitForExpectationsWithTimeout:10.0 handler:nil];
```

Comment:
<img width="85" alt="image" src="https://user-images.githubusercontent.com/139364/98627604-73b5d780-22c9-11eb-8d1b-5c949b597d99.png">

we're gonna need to figure this out eventually

Reply:
Yeah... I'm assuming these are failing simply because the CI machines are slower than my laptop. But definitely seems strange. 😬